tesseract_common/include/tesseract_common/utils.h

```diff
@@ -368,5 +368,30 @@ inline bool almostEqualRelativeAndAbs(double a,
   return (diff <= largest * max_rel_diff);
 }
 
+/**
+ * @brief Check if two Eigen VectorXd are relatively equal
```

Comment:
This function is checking both absolute and relative equality, not just relative equality, right?

Reply:
That is correct. I will update the description.